README.md

```diff
@@ -35,8 +35,6 @@ The goal of this project is to develop the official Corona-Warn-App for Germany
 
 * The Exposure Notification API is going to block you from successfully testing the Application unless you are whitelisted inside GMS; **Shoutout to @pocmo for working on a demo mode to test unreleased app versions in advance without whitelisting (issue [#321](https://github.com/corona-warn-app/cwa-app-android/issues/321))**, if you want to contribute you can reach out [here](https://github.com/pocmo/cwa-app-android) for the time being - thank you!
 * The Storage is currently based on Encrypted Shared Preferences and SQL Cipher (SQLite) - this could change in the future
-* Test Coverage is generally low and needs to be improved. We appreciate your help here!
-* In General every TODO comment within the code or the documentation can be regarded as an issue. You are free to tackle the TODOs anytime!
```

Comment:
As per tools , that is still true, sonarcube  will complain if there is a TODO in the code ;)

Reply:
For sure. But are there TODOs in the PROD versions?

Reply:
Yes, it is in the code

Reply:
Okay, didn't know this, but do you want outside contributors to tackle them?

Reply:
If they want to , I think it is mentioned that way if someone want to contribute in the App, it is easier to pick one of the TODOs and tackle it.

Reply:
Okay, I will revert the change to the TODO section, but the thing with the test coverage is fine, or?

Reply:
I would keep it , it tells us how much technical debt we have

Reply:
But maybe change the wording, as test coverage is not low, or?

Reply:
I don't get it, what do you mean by ` as test coverage is not low` ?

Reply:
The text says that test coverage is generally low. I deleted this text. Now you said you would like to keep it. But is it still true that test coverage is generally low?

Reply:
Yes, ~ 43 % could be better see 
<img width="755" alt="Screenshot 2022-03-31 at 13 34 56" src="https://user-images.githubusercontent.com/25054729/161046098-5b9a37f0-ed85-4cbe-baac-f7688dc33af8.png">

Reply:
Okay then I'll close this PR. Leaving a link to why I opened this PR here: https://twitter.com/marcoh___/status/1507272525813854209?s=21&t=H3gDI0ftoZ_vsjNiSUcyFA

Reply:
With every build, you can see such information in the report, https://sonarcloud.io/project/issues?id=corona-warn-app_cwa-app-android&branch=main&resolved=false&types=CODE_SMELL

 - You can scroll down to TODOs, and the plan is to tackle those issues in the transition period

Reply:
Okay, thanks Mohamed!

What exactly do you mean by transition period? Transition from what?

Reply:
I mean when there is no new features coming ,but focus on bug fixing